src/rules/valid-expect-in-promise.ts

```diff
@@ -110,7 +110,8 @@ const isPromiseMethodThatUsesValue = (
     }
 
     if (
-      ['Promise.resolve', 'Promise.reject'].includes(nodeName as string) &&
+      nodeName &&
```

Comment:
This check is arguably more expensive at runtime - it's very minor, but I'd rather keep the cast

Reply:
Got it. I realized I missed considering performance aspects 🙇‍♂️
 I've addressed it in commit b030cc7.